rock.osdeps

```diff
@@ -274,6 +274,7 @@ libproc:
     ubuntu:
         '10.04,10.10,11.04,11.10,12.04': libproc-dev
         '12.10,13.04,13.10': libprocps0-dev
+        '16.04': libprocps4-dev
```

Comment:
How the version vs. default entries work is that the 'default' entry should point to the latest Ubuntu (as it is the most likely package name for version N+1). If the default changes, you have to add all the entries in-between, here:

```
'14.04,14.10,15.04,15.10': libprocps3-dev
default: libprocps4-dev # 16.04 and later ... until the next package name change
```